### ISSUE TITLE
[bug]

### ISSUE THREAD

**Issue by egenomics** [bactopia/bactopia#359]
**Description**
Hi, 
I am having problems downloading the datasets for the tutorial. I have tried the regular bactopia datasets and also restricting to --ariba card (see below) but both apporaches fail.
Is there a known problem for the database downloads?

**Steps to Reproduce**
```
singularity exec -B /well/buck/users/xhs232/:/well/buck/users/xhs232/ /well/buck/users/xhs232/soft/containers/bactopia_latest.sif bactopia datasets --ariba card

2022-10-03 17:05:46:root:INFO - Setting up Ariba datasets
2022-10-03 17:05:46:root:INFO - Setting up card Ariba dataset
External command failed with exit code 1!

Command:
bash -c 'ariba getref card card'

Standard output:
Getting available CARD versions
Downloading "https://card.mcmaster.ca/download" and saving as "download.html" ...Traceback (most recent call last):
  File "/opt/conda/envs/bactopia/bin/ariba", line 312, in <module>
    args.func(args)
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/tasks/getref.py", line 11, in run
    getter.run(options.outprefix)
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/ref_genes_getter.py", line 664, in run
    exec('self._get_from_' + self.ref_db + '(outprefix)')
  File "<string>", line 1, in <module>
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/ref_genes_getter.py", line 78, in _get_from_card
    versions = self._get_card_versions('download.html')
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/ref_genes_getter.py", line 44, in _get_card_versions
    common.download_file('https://card.mcmaster.ca/download', tmp_file, max_attempts=self.max_download_attempts, sleep_time=self.sleep_time, verbose=True)
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/common.py", line 71, in download_file
    raise Error('Error downloading: ' + url)
ariba.common.Error: Error downloading: https://card.mcmaster.ca/download

Standard error:
Getting available CARD versions
Downloading "https://card.mcmaster.ca/download" and saving as "download.html" ...Traceback (most recent call last):
  File "/opt/conda/envs/bactopia/bin/ariba", line 312, in <module>
    args.func(args)
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/tasks/getref.py", line 11, in run
    getter.run(options.outprefix)
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/ref_genes_getter.py", line 664, in run
    exec('self._get_from_' + self.ref_db + '(outprefix)')
  File "<string>", line 1, in <module>
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/ref_genes_getter.py", line 78, in _get_from_card
    versions = self._get_card_versions('download.html')
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/ref_genes_getter.py", line 44, in _get_card_versions
    common.download_file('https://card.mcmaster.ca/download', tmp_file, max_attempts=self.max_download_attempts, sleep_time=self.sleep_time, verbose=True)
  File "/opt/conda/envs/bactopia/lib/python3.6/site-packages/ariba/common.py", line 71, in download_file
    raise Error('Error downloading: ' + url)
ariba.common.Error: Error downloading: https://card.mcmaster.ca/download

```

**Expected Behavior**
Correct download of dataset.

**Execution Environment**

- Bactopia Version: bactopia 1.5.5
- OS: Ubuntu
- Environment: singularity container

**Additional Information**
none

**Comment by rpetit3** [comment]
Hi @egenomics,

It might be worth updating to the latest version of Bactopia, and making use of the RGI tool (https://bactopia.github.io/v2.1.1/bactopia-tools/rgi/). 

In recent versions of Bactopia, I've removed Ariba from `bactopia datasets` due to similar issues as you are having now. Instead Ariba is now available as a Bactopia Tool (https://bactopia.github.io/v2.1.1/bactopia-tools/ariba/).

If you have to stay on v1.5.5, there was a fix to Ariba related to CARD downloads (https://github.com/sanger-pathogens/ariba/pull/302), I wonder if that singularity container has a version of Ariba without the fix.

Cheers,
Robert

**Comment by egenomics** [comment]
Hi,
I see! I will download the latest singularity container. I have noticed that in the documentation it links to the v1.5.5 version:
https://bactopia.github.io/installation/

Best,
JL

**Comment by rpetit3** [comment]
Thank you for bringing that to my attention, I need to update that.

Can use this one: `quay.io/bactopia/bactopia:2.1.1` or to download image directly from the Galaxy Project: https://depot.galaxyproject.org/singularity/bactopia%3A2.1.1--hdfd78af_0

**Comment by rpetit3** [comment]
Hi @egenomics 

I've released [v2.2.0](https://github.com/bactopia/bactopia/releases/tag/v2.2.0), going to go ahead and close this.

Cheers,
Robert
